skbio/stats/composition.py

```diff
@@ -608,10 +611,269 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0-dev")
+def ancom(table, groups,
+          alpha=0.05,
+          tau=0.02,
+          theta=0.1,
+          multiple_comparisons_correction=None,
+          significance_test=scipy.stats.ttest_ind):
+    r""" Performs a differential abundance test using ANCOM
+
+    This is done by calculating pairwise log ratios between all features
+    and performing a signficance test to determine if there is a
+    significant difference in feature ratios with respect to the
+    variable of interest.
+
+    In an experiment with only two treatments, this test tests the
+    following hypothesis for feature :math:`i`
+
+    .. math::
+        H_{0i}:\mathbb{E}[\ln(u_i^{(1)})] = \mathbb{E}[\ln(u_i^{(2)})]
+
+    where :math:`u_i^{(1)}` is the mean abundance for feature
+    :math:`i` in the first group and :math:`u_i^{(2)}` is the
+    mean abundance for feature :math:`i` in the second group.
+
+    This method can be extended to an arbitrary number of classes
```

Comment:
Could you have the function default to a one-way ANOVA instead so it works with 2 or more groups? AFAIK a one-way ANOVA is a generalization of the t-test, so t-test and ANOVA should give the same results when there are 2 groups. That way users won't need to pass a different function when they have more than 2 groups.

**I'm not a statistician, someone please verify this!**